### ISSUE TITLE
The admin program does not seem to be able to publish messages

### ISSUE THREAD

**Issue by petten** [darrenjs/wampcc#13]
I believe the only difference between call and publish is command line options -c and -p, with the other arguments basically specified the same ways.

-c works for me, but -p does not seem to.

I have called provide() for -c, and subscribe() for -p, both with logs from within wampcc indicating positive outcomes ("procedure registered" and "topic subscribed to", respectively).

**Comment by petten** [comment]
Well, I use bonefish as the router, where the real issue could be.

**Comment by darrenjs** [comment]
hi .... can you write the full command line that you are using?   Also, with admin command, try the `-d -d` option to view the actual bytes written to the socket.

**Comment by petten** [comment]
I took the wampcc from the master some time after 1.3.1 was released. (So it's somewhere in between release 1.3.1 and the next release, but probably closer to 1.3.1.) The version of admin does not seem to support -d -d. Nor does it seem to support the ws://... scheme; instead, it requires host and port to be specified as two separate arguments.

The -c option (that makes a call to my wampcc callee) has worked for me from the very beginning. Now that I added some code to handle published messages, the only change I made to the admin command line was replacing the -c option with a -p one, with other arguments staying about the same (except for necessary adjustments such as the payload of the --arglist).

**Comment by darrenjs** [comment]
hi ... yes, the ws:// and -d were added sometime after the 1.3.1 release.  Recommend you checkout the master and use that version.

**Comment by petten** [comment]
I just downloaded and built the current master.

I took only the newly built `admin` program, and ran it against my present setup.

Guess what? It worked!
- It's able to publish messages via the `bonefish` router.
- The `-d -d` logging works, with plenty of lines showing up on the console.
- And my new subscriber code (though based on the older `wampcc` release) worked right away as expected.

For now, I'm keeping the setup/old code base as is. I'm sure I'll have to update to the latest `wampcc` version, but when that'll happen depends.

Many thanks to you Darren for the active support!

I'm closing the issue now.
